tests/features/bootstrap/FeatureContext.php

```diff
@@ -75,4 +75,19 @@ public function iCreateColorTerm() {
     $element->findButton('Save')->click();
   }
 
+  /**
+   * Creates a menu item with specified name in the specified menu.
+   *
+   * @Given /^I create an item "([^"]*)" in the "([^"]*)" menu$/
+   */
+  public function iCreateItemInTheMenu($menu_item, $menu_name) {
+    $path = '/admin/structure/menu/manage/' . $menu_name . '/add';
+    $this->getSession()->visit($this->locatePath($path));
+    $element = $this->getSession()->getPage();
+    $element->fillField('Menu link title', $menu_item);
+    $element->fillField('Link', 'http://example.com');
+    $element->checkField("Show as expanded");
+    $element->findButton('Save')->click();
```

Comment:
I think this should simplify maintenance. Compare:
Method with repeating steps in multiple features will require changing multiple lines in multiple scenarios.
Method with reusable function will require change in one place.

Also this function is built on pure behat core and should be supported with upgrades.